topics/protocol.md

```diff
@@ -235,7 +235,7 @@ specify a Null value (usually the Null Bulk String is used, but for historical
 reasons we have two formats).
 
 For instance when the `BLPOP` command times out, it returns a Null Array
-that has a count of `-1` as in the following example:
+that has a count of "-1" as in the following example:
```

Comment:
Here -1 is just a number. It's not a string. I think quotes is misleading in this case. I think no formatting at all is OK. Backticks is also OK.

Reply:
yes. i just wanted to quotes it
![image](https://user-images.githubusercontent.com/22811481/148928927-8fb44e97-8c6d-4cd5-8cf5-188a3a20be96.png)

Reply:
I think quotes is misleading, since this text refers to the number -1, not the string "-1".

Reply:
right, i decided to just use the number 1